openrouteservice/src/test/java/heigit/ors/api/requests/matrix/MatrixRequestHandlerTest.java

```diff
@@ -31,8 +32,8 @@
     private Coordinate coordinate2 = new Coordinate();
     private Coordinate coordinate3 = new Coordinate();
 
-    private List<List<Double>> maximumLocations;
     private List<List<Double>> minimalLocations;
+    private int maximumLocationsCombo;
```

Comment:
if we are still dealing with the maximum routes, then the variable should be named using Routes and not Locations